packages/coinstac-simulator/src/db-server.js

```diff
@@ -5,58 +5,126 @@
  * @module db-server
  */
 
-require('./handle-errors')();
-const pdbs = require('spawn-pouchdb-server');
-const cp = require('child_process');
-const pdbsConfig = require('./.pouchdb-server-config');
+require('./utils/handle-errors');
+
 const cloneDeep = require('lodash/cloneDeep');
+const common = require('coinstac-common');
+const config = require('./utils/config');
```

Comment:
Second thought, I think its fine as is, we can revisit if more stuff gets tossed in